package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.13.0",
   "description": "A minimal node SOAP client",
   "engines": {
-    "node": ">=0.8.0"
+    "node": ">=0.10.0"
```

Comment:
I think we should it mark clearly within the next release notes, that we've dropped the support for node 0.8 with this commit.